iroha2/data_model/account/__init__.py

```diff
@@ -7,41 +7,43 @@
 )
 from ..domain import Id as DomainId
 from ..isi import Registrable
+from ...crypto import PublicKey
+from .. import wrapper
 
 
+@wrapper(_Account)
 class Account(_Account, Registrable):
 
     def __init__(self,
                  id: Union[Id, str],
                  assets: Optional[dict] = None,
-                 permission_tokens: Optional[list] = None,
+                 signatories: Optional[list[PublicKey]] = None,
                  metadata: Optional[dict] = None,
                  roles: Optional[list] = None):
 
         if assets is None:
```

Comment:
This could be done with a single `__setattr__` in a loop, but the more explicit style can stay for now.